views/home.html

```diff
@@ -2,20 +2,17 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Gno.land</title>
+    <title>Gno.land - {{ .Data.Title }}</title>
+    <meta name="description" content="{{ .Data.Description }}" />
     {{ template "html_head" }}
   </head>
   <body onload="main()">
     <div id="root">
-      <div id="header">
-        {{ template "header_logo" }}
-        {{ template "header_buttons" }}
-      </div>
-      <div id="home">
-        <pre id="source">
-{{ .Data.MainContent }}
-        </pre>
+      <div id="header">{{ template "header_logo" }} {{ template "header_buttons" }}</div>
```

Comment:
not sure if what was desired was {{- -}} or not, I think {{- -}} removes preceding and trailing whitespaces if i remember correctly. anyways if you already knew that, never mind.

Reply:
You are right (if it's same as in Hugo framework).

Reply:
yeah, it's the `html/template` golang package; hugo is using it extensively and extends the builtin functions.

it's a gem from Golang which is super simple and powerful for code/page/whatever generation.